ml-agents/mlagents/trainers/ghost/trainer.py

```diff
@@ -301,13 +301,22 @@ def end_episode(self):
     def save_model(self, name_behavior_id: str) -> None:
         """
         Forwarding call to wrapped trainers save_model
+        Loads the latest snapshot then reloads current snapshot.
```

Comment:
Can we make the comment "Loads the latest policy weights, saves it, then reloads the current policy weights before resuming training." Just so we don't forget exactly what this code does 3 months from now